src/views/Correction.vue

```diff
@@ -741,11 +708,41 @@ export default class Correction extends Mixins(
     }
   }
 
-  /** Handler for dialog Exit click events. */
-  navigateToDashboard (ignoreChanges: boolean = false): void {
-    if (ignoreChanges) this.haveChanges = false
-    this.$router.push({ name: Routes.DASHBOARD })
-      .catch(() => {}) // ignore error in case navigation was aborted
+  /**
+   * Routes to dashboard if there are no outstanding changes,
+   * else prompts user before routing.
+   */
+  goToDashboard (force: boolean = false): void {
+    // check if there are no data changes
+    if (!this.haveChanges || force) {
+      // route to dashboard
+      this.$router.push({ name: Routes.DASHBOARD })
+        .catch(() => {}) // ignore error in case navigation was aborted
+      return
+    }
+
+    // open confirmation dialog and wait for response
+    this.$refs.confirm.open(
+      'Unsaved Changes',
+      'You have unsaved changes in your Change of Directors. Do you want to exit your filing?',
```

Comment:
is text correct here?

Reply:
Fixing this too. And also Annual Report.